.github/workflows/integrationTest.yml

```diff
@@ -499,4 +557,57 @@ jobs:
         run: terraform init
 
       - name: Terraform destroy
-        run: terraform destroy --auto-approve
\ No newline at end of file
+        run: terraform destroy --auto-approve
+
+  ECSFargateLinuxIntegrationTest:
+    name: 'StopLocalStack'
```

Comment:
I don't think that this is the right name for the job

Reply:
Right.Changed to only ECSFargate though with the newest commit.